tables/superdarn/radar.dat

```diff
@@ -32,44 +32,44 @@
-20  1 20100122 25000101 "McMurdo"                 "Penn State University"               "hdw.dat.mcm" "mcm" "m"
-21  1 20100214 25000101 "Falkland Islands"        "British Antarctic Survey"            "hdw.dat.fir" "fir" "q"
-22  1 20130122 25000101 "South Pole Station"      "Penn State University"               "hdw.dat.sps" "sps"
-24  1 20141201 25000101 "Buckland Park"           "La Trobe University"                 "hdw.dat.bpk" "bpk"
-32  1 20050610 25000101 "Wallops Island"          "JHU/APL"                             "hdw.dat.wal" "wal" "i"
-33  1 20080202 25000101 "Blackstone"              "Virginia Tech"                       "hdw.dat.bks" "bks"
-40  1 20061101 25000101 "Hokkaido East"           "Nagoya University"                   "hdw.dat.hok" "hok"
-41  1 20141029 25000101 "Hokkaido West"           "Nagoya University"                   "hdw.dat.hkw" "hkw"
-50  1 20190625 25000101 "Jiamusi East"            "National Space Science Center, CAS"  "hdw.dat.jme" "jme"
-64  1 20080101 25000101 "Inuvik"                  "University of Saskatchewan"          "hdw.dat.inv" "inv"
-65  1 20070501 25000101 "Rankin Inlet"            "University of Saskatchewan"          "hdw.dat.rkn" "rkn"
-66  1 20120806 25000101 "Clyde River"             "University of Saskatchewan"          "hdw.dat.cly" "cly"
-90  1 20161019 25000101 "Longyearbyen"            "University Centre in Svalbard"       "hdw.dat.lyr" "lyr"
-96  1 20130123 25000101 "Dome C East"             "IAPS-INAF"                           "hdw.dat.dce" "dce"
-97  1 20190129 25000101 "Dome C North"            "IAPS-INAF"                           "hdw.dat.dcn" "dcn"
-204 1 20091120 25000101 "Fort Hays West"          "Virginia Tech"                       "hdw.dat.fhw" "fhw"
-205 1 20091120 25000101 "Fort Hays East"          "Virginia Tech"                       "hdw.dat.fhe" "fhe"
-206 1 20101104 25000101 "Christmas Valley West"   "Dartmouth College"                   "hdw.dat.cvw" "cvw"
-207 1 20101104 25000101 "Christmas Valley East"   "Dartmouth College"                   "hdw.dat.cve" "cve"
-208 1 20120906 25000101 "Adak West"               "Penn State University"               "hdw.dat.adw" "adw"
-209 1 20120906 25000101 "Adak East"               "Penn State University"               "hdw.dat.ade" "ade"
-512 0 20110101 25000101 "EKB Radar"               "Russian Academy of Sciences"         "hdw.dat.ekb" "ekb"
+0    0 20080203 25000101 "Test Radar"              "United Nations of SuperDARN"         "hdw.dat.tst" "tst" "x"
+1    1 19831001 25000101 "Goose Bay"               "Virginia Tech"                       "hdw.dat.gbr" "gbr" "g"
+2   -1 19861001 19950930 "Schefferville"           "CNRS/LPC2E"                          "hdw.dat.sch" "sch" "s"
+3    1 19930901 25000101 "Kapuskasing"             "Virginia Tech"                       "hdw.dat.kap" "kap" "k"
+4   -1 19880101 20160119 "Halley"                  "British Antarctic Survey"            "hdw.dat.hal" "hal" "h"
```

Comment:
Should this be `20160120` to still allow plotting of the FOV on `20160119`?  I'm not actually sure of the software behavior if the `hdw.dat` entry extends a few hours past the `radar.dat` end date.

Reply:
Good catch. I did a quick test with `fov_plot`. It works with `-d 20160119 -t 00:00` but not `-d 20160119 -t 00:01`.

Update on the way...